services/slam/client_test.go

```diff
@@ -270,11 +318,58 @@ func TestClientFailingService(t *testing.T) {
 		test.That(t, im, test.ShouldBeNil)
 		test.That(t, pc.PointCloud, test.ShouldBeNil)
 
+		// test get pointcloud map stream
+		fullBytes, err := slam.GetPointCloudMapFull(context.Background(), failingSLAMClient, nameFail)
```

Comment:
[q] do we want to move to a single context (everywhere except the context cancel test) instead of creating a new background context each time as we have in the past?

Reply:
Note: if yes, I can also move the cancel context check to the end and cancel it then